src/components/builder/derive-address.tsx

```diff
@@ -0,0 +1,25 @@
+import {FormattedMessage, useIntl} from 'react-intl';
+
+import {Checkbox} from '../formio';
+
+const DeriveAddress = () => {
+  const intl = useIntl();
+  const tooltip = intl.formatMessage({
+    description: "Tooltip for 'DeriveAddress' builder field",
+    defaultMessage: 'Derive address.',
+  });
+  return (
+    <Checkbox
+      name="disabled"
```

Comment:
that doesn't seem right, the checkbox controls the value of the `component.deriveAddress` property, so:

```suggestion
      name="deriveAddress"
```